### ISSUE TITLE
go install github.com/mikefarah/yq/v4@latest

### ISSUE THREAD

**Issue by aertoria** [coinbase/chainstorage#57]
go install github.com/mikefarah/yq/v4@latest 
required1.20 at least. and is incompatible with 1.18 as it's been deprecated

**Comment by leozc** [comment]
@bestmike007  is this what you talked about in an incoming check-in?

cc: @jiezhang

**Comment by bestmike007** [comment]
Yes, I was trying to run CI with multiple go versions. https://github.com/bestmike007/chainstorage/actions/runs/6671190865

This issue can be get around with by installing the binary directly from https://github.com/mikefarah/yq/releases/tag/v4.35.2 or with a toolchain manager like asdf: https://github.com/coinbase/chainstorage/pull/42/files#diff-ad8a239f2d9110a9929810a189865ad0515e31355df3088588fccbb94879ce4fR15
